spec/workers/scheduler/accounts_statuses_cleanup_scheduler_spec.rb

```diff
@@ -101,6 +101,12 @@
   end
 
   describe '#perform' do
+    around do |example|
+      Timeout.timeout(60) do
```

Comment:
Is the background here that this has intermittent failure issues or otherwise gets stuck in spec runs, and you want to kill it when it does?

Reply:
No, the background is that there's a test for a case where it was once an infinite loop.